packages/kurt-open-ai/spec/generateStructuredData.spec.ts

```diff
@@ -16,4 +17,37 @@ describe("KurtOpenAI generateStructuredData", () => {
     )
     expect(result.data).toEqual({ say: "hello" })
   })
+
+  test("throws a validate error from an impossible schema", async () => {
```

Comment:
This test is failing because `/kurt-open-ai/spec/snapshots/KurtOpenAI_generateStructuredData_throws_a_validate_error_from_an_impossible_schema.yaml` is missing